### PR TITLE
Update komodo from 11.1.1 to 12.0.1

### PR DESCRIPTION
**Pull request by uberhacker**
Komodo IDE is now free.  See https://www.activestate.com/products/komodo-ide/.  Tested on x86_64.

### CODE REVIEW

File: packages/komodo.rb [hunk 1]
```diff
@@ -26,12 +22,27 @@ class Komodo < Package
...
-    system './install.sh'
+    system "./install.sh -I #{CREW_DEST_PREFIX}"
+  end
+
+  def self.remove
+    print "Would you like to remove the config directories? [y/N] "
+    response = STDIN.getc
+    config_dirs = ["#{HOME}/.komodoide", "#{HOME}/.activestate"]
+    config_dirs.each { |config_dir|
+      if Dir.exists? config_dir
```
Comment (supechicken) [inline packages/komodo.rb:37]: This line can be removed as `rm_rf` will not produce error when file not exist

**Comment by saltedcoffii** [comment]
Does the EULA permit us to distribute binaries? I would hate for chromebrew  to get in trouble

**Comment by satmandu** [comment]
@saltedcoffii has a valid concern here. I'm not seeing anything in the EULA on the web site about allowing distribution of binaries: https://www.activestate.com/products/komodo-ide/license-agreement/

**Comment by uberhacker** [comment]
I think the main concern of the license is to restrict selling or reverse engineering the software.  We are not doing any of that.  It would be to ActiveState's benefit that we make their software easily available on Chromebooks because it promotes more users to sign up for subscriptions.

File: packages/komodo.rb (hunk 1 above)
Comment (uberhacker) [reply packages/komodo.rb:37]: True but if the directory doesn't exist then the command is executed unnecessarily.

**Comment by satmandu** [comment]
@uberhacker Respectfully, I'm not sure that's how licensing works when the lawyers get involved. The conservative approach would be to only have binaries that have a clear indication (through licensing or otherwise) that distribution is allowed.

If there's a shadow of a doubt... don't distribute it.

**Comment by uberhacker** [comment]
Can you point out the specific paragraph in the license that casts a shadow of doubt because I'm not seeing it?

**Comment by satmandu** [comment]
Screenshot 2021-06-30 11 56 12

**Pull request merged**